src/slic3r/GUI/ObjectDataViewModel.cpp

```diff
@@ -558,8 +558,12 @@ void ObjectDataViewModel::UpdateBitmapForNode(ObjectDataViewModelNode *node)
         std::vector<wxBitmap> bmps;
         if (node->has_warning_icon())
             bmps.emplace_back(node->warning_icon_name() == WarningIcon ? m_warning_bmp : m_warning_manifold_bmp);
+        if (node->has_warning_icon() && node->has_lock()) // ORCA: Add spacing between icons if there are multiple
```

Comment:
The code would be simpler if we move these two lines in the next `if` block

if (node->has_lock()) {
    if(!bmps.empty())
        bmps.emplace_back(create_scaled_bitmap("dot", nullptr, int(wxGetApp().em_unit() / 10) * 4));
    bmps.emplace_back(m_lock_bmp);
}